frontend/public/components/group.tsx

```diff
@@ -0,0 +1,216 @@
+import * as React from 'react';
+import { match } from 'react-router-dom';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+
+import { GroupModel, UserModel } from '../models';
+import { referenceForModel, GroupKind, K8sKind } from '../module/k8s';
+import { DetailsPage, ListPage, Table, TableRow, TableData } from './factory';
+import { addUsersModal, removeUserModal } from './modals';
+import { RoleBindingsPage } from './RBAC';
+import { fromNow } from './utils/datetime';
+import {
+  asAccessReview,
+  EmptyBox,
+  Kebab,
+  KebabAction,
+  KebabOption,
+  navFactory,
+  ResourceKebab,
+  ResourceLink,
+  ResourceSummary,
+  SectionHeading,
+} from './utils';
+
+const addUsers: KebabAction = (kind: K8sKind, group: GroupKind) => ({
+  label: 'Add Users',
+  callback: () =>
+    addUsersModal({
+      resourceKind: kind,
```

Comment:
Looking at the addUsersModal implementation, it doesn't seem like this prop is used. I might be missing something though.

Reply:
Yeah, this was an oversight. Fixed. We should probably add a generic type to `createModalLauncher` to catch stuff like this